client/state/difm/selectors.ts

```diff
@@ -0,0 +1,6 @@
+import 'calypso/state/difm/init';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export function getDIFMState( state: { difm: any } ) {
```

Comment:
We should probably start building a partial state typescript model without relying on any.

Simply we can add in a state called `DifmState` and include in  state/types.ts